src/hotspot/share/opto/superword.cpp

```diff
@@ -2790,6 +2919,17 @@ bool SuperWord::output() {
         return false; // bailout
       }
 
+#ifdef ASSERT
+      // Mark Load/Store Vector for alignment verification
+      if (AlignVector && VerifyAlignVector && cl->is_main_loop()) {
```

Comment:
`AlignVector &&` no need if you do check when VerifyAlignVector is set.

Reply:
Done